probes/webhooksWithoutTokenAuth/def.yml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: webhooksWithoutTokenAuth
```

Comment:
I think it's called a "secret", not a "token", at least on GitHub https://docs.github.com/en/webhooks/using-webhooks/creating-webhooks. If we use "secret", let's also update the rest of the text accordingly